docs/modules/ROOT/pages/spring-cloud-commons/common-abstractions.adoc

```diff
@@ -135,7 +135,7 @@ Please see the documentation of the `ServiceRegistry` implementation you use for
 For instance, Eureka's supported statuses are `UP`, `DOWN`, `OUT_OF_SERVICE`, and `UNKNOWN`.
 
 [[rest-template-loadbalancer-client]]
-== Spring RestTemplate as a Load Balancer Client
+== Spring `RestTemplate` as a LoadBalancer Client
```

Comment:
Remove the ticks around `RestTemplate`. Your instinct is right, but we have had trouble with rendering code fonts in headings, so we had to remove it.